src/Tools/McpServer/Helpers/ToolOutputHelper.cs

```diff
@@ -0,0 +1,97 @@
+// ------------------------------------------------------------------------
+// This file is licensed to you under the MIT License.
+// ------------------------------------------------------------------------
+
+using System.Globalization;
+using System.Text;
+using Microsoft.FluentUI.AspNetCore.McpServer.Models;
+
+namespace Microsoft.FluentUI.AspNetCore.McpServer.Helpers;
+
+/// <summary>
+/// Helper methods for formatting MCP tool output.
+/// </summary>
+internal static class ToolOutputHelper
+{
+    /// <summary>
+    /// Truncates a summary to the specified maximum length.
+    /// </summary>
+    public static string TruncateSummary(string? summary, int maxLength)
+    {
+        if (string.IsNullOrEmpty(summary))
+        {
+            return "-";
+        }
+
+        if (summary.Length <= maxLength)
+        {
+            return summary;
+        }
+
+        return summary[..(maxLength - 3)] + "...";
+    }
+
+    /// <summary>
+    /// Checks if a parameter name is common enough to include in examples.
+    /// </summary>
+    public static bool IsCommonExampleParam(string paramName)
```

Comment:
Why to you have this method and all methods below? The single method used is `TruncateSummary`. All others are only in Unit Tests.

Reply:
Old code from a previous integration. Removed.